regtests/bin/run_cmake_test

```diff
@@ -2023,7 +2023,6 @@ do
 done # end of loop on progs
 
 # 3.i Track output ---------------------------------------------------------- #
-
 case $outopt in
   native) out_progs="ww3_trck" ;;
   netcdf) out_progs="ww3_trnc" ;;
```

Comment:
@ukmo-ccbunney i just figured out why we don't have output here, we use the option "-o all" in the matrix for ncep https://github.com/NOAA-EMC/WW3/blob/develop/regtests/bin/matrix_ncep#L180-L185 and there fore we get in the "out_progs=""" section.

Reply:
That was the non-cmake option but cmake is the same: https://github.com/NOAA-EMC/WW3/blob/develop/regtests/bin/matrix_cmake_ncep#L177

Reply:
I'll add an enrty for `output="all"` for the track output section in this PR if you like (seems like a related issue not warranting a new PR?)

Reply:
That's be great @ukmo-ccbunney ! Seems related to me as well.  Thanks so much for your help with this!